### PR TITLE
i did my first landing

### PR DESCRIPTION
**Pull request by RomSmile**
DEMO LINK: https://RomSmile.github.io/layout_miami/

### CODE REVIEW

File: src/styles/blocks/header.scss [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  &__call {
+    text-decoration: none;
+    
+    @include onDesktop() {
+      &::before {
+        content: "+1 123 555-55-55";
+        position: relative;
+        top: 50%;
+        transform: translate(-100%, 50%);
+
```
Comment (Serhii-Usenko) [inline src/styles/blocks/header.scss:29]: its center but u have it https://prnt.sc/1eesm23

File: src/styles/blocks/form-field.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    border-color: $colorBlue;
+    outline: none;
+  }
+
+  &--textarea {
+    height: 144px;
+    resize: none;
+    font-family: inherit;
+  }
+}
```
Comment (Serhii-Usenko) [inline src/styles/blocks/form-field.scss:32]: U dont have "message" https://prnt.sc/1eet21c

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,397 @@
...
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="place__description">
+              <a href="" class="place__link">Edition Residens </a>
+              / Miami Beach, FL 33139
```
Comment (Serhii-Usenko) [inline src/index.html:189]: no link

**Review by Serhii-Usenko — COMMENTED** [review]
excellent

**Comment by danheim** [comment]
https://www.loom.com/share/89999b713a864e56a572c8e1f7cf9f27
use google fonts to import Roboto

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]
https://www.loom.com/share/b0337d39b4b8425d9cd0d58b2400e324

**Review by Denchepornyuk — APPROVED** [review]

**Pull request closed without merging**